docs/user/openstack/README.md

```diff
@@ -69,6 +69,25 @@ In addition, it covers the installation with the default CNI (OVNKubernetes), as
 
 ## OpenStack Requirements
 
+Before starting to install OpenShift, ensure following OpenStack services are
+available, as there are couple of them which we are relying on:
+
+- Keystone
+- Neutron
+- Nova
+  - with Metadata service enabled
+- Placement
+- Glance
+- Storage solution for the image registry:
+  - Swift
+  - Cinder with dynamically-provisioned volumes
```

Comment:
`with dynamically-provisioned volumes` is irrelevant here.

Reply:
ack